sql/parser/builtins.go

```diff
@@ -331,12 +334,23 @@ var builtins = map[string][]builtin{
 			types:      typeList{},
 			returnType: DummyFloat,
 			impure:     true,
-			fn: func(args DTuple) (Datum, error) {
+			fn: func(_ EvalContext, args DTuple) (Datum, error) {
 				return DFloat(rand.Float64()), nil
 			},
 		},
 	},
 
+	"unique_id": {
```

Comment:
I've added another commit which renames `unique_id()` to `unique_bytes()` and adds a `unique_int()` builtin. The implementation of `unique_bytes()` changed so that it is now `nanoseconds+node-id` (no randomness). Might be overkill to have both `unique_bytes()` and `unique_int()`. PTAL.